python/lsst/daf/persistence/posixStorage.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, uri):
         :return:
         """
         self.log = Log.getLogger("daf.persistence.butler")
-        self.root = urllib.parse.urlparse(uri).path
+        self.root = self._pathFromURI(uri)
         if self.root and not os.path.exists(self.root):
             os.makedirs(self.root)
```

Comment:
I looked at `safeMakeDir` and I'm not sure if it is actually safe if the path it is given contains multiple [new] nested dirs.